### PR TITLE
Update npm package details, license, changelog and CI config

### PR DESCRIPTION
**Pull request by pdcastro**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+   incurred by, or claims asserted against, such Contributor by reason
+   of your accepting any such warranty or additional liability.
+
+END OF TERMS AND CONDITIONS
+
+--------------------------------------------------------------------------
+
+The MIT License (MIT)
+
+Copyright (c) 2018 Zeit, Inc.
```
Comment (pdcastro) [inline LICENSE.md:183]: This "dual licensing" Apache 2.0 + MIT was agreed in the architecture call of 23 Apr 2020. It satisfies the legal requirements of the original repo (https://github.com/zeit/dockerignore/blob/0.0.5/license.md) and balena's own licensing policy (Apache 2.0) and balena CI checks. The "Balena Ltd." name will be mentioned in the source code files themselves.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Appveyor CI configuration
+# https://www.appveyor.com/docs/
+
+branches:
+  only:
+    - master
+
+build: off
+test: off
```
Comment (pdcastro) [inline appveyor.yml:9]: This will be turned on for Windows testing in a subsequent PR.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@zeit/dockerignore",
+  "name": "@balena/dockerignore",
   "version": "0.0.5",
-  "description": "dockerignore is a manager and filter for .dockerignore rules and is a fork of ignore.",
+  "description": "dockerignore is a file filter library compatible with Docker and the node-ignore API",
```
Comment (pdcastro) [inline package.json:4]: Reasoning:
* This package's name is already `dockerignore`, so 2 further occurrences of `dockerignore` in the description line seemed redundant.
* While I can understand the meaning of "manager and filter" given all the context I've got, I thought it was rather vague/generic to a developer doing a package search -- especially the "manager" term.
* The good thing about the expression "a fork of ignore" is that it clarifies that "ignore" is a repo (although the github repo is actually called "node-ignore"), but I thought it didn't answer the question _"so what?"_ ("a fork of ignore" - ok, so what?...) Many users won't have knowledge of the "ignore" package, even though it's popular on `npm`.
* My proposed description is not perfect either, but I thought the advantages were:
  * It includes the words "Docker" and "node", which are widely known.
  * "compatible with the node-ignore API" hints an answer to the "so what" question: API compatibility. Granted, people won't know what "the node-ignore API" is, but:
    * They will be left with the impression that "node-ignore API" is a thing they should know about.
    * If they google "node-ignore", they get the right answers as the top 3 results! Whereas if they google just "ignore," they get too much "noise".
  * Experienced Node.js developer will recognise the pattern `node-xxx` as the Github repo corresponding to the `npm` `xxx` package, hence `node-ignore` also hints at an `npm` package called `ignore`.

File: .resinci.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+npm:
+  platforms: []
```
Comment (pdcastro) [inline .resinci.yml:3]: A subsequent PR will enable Linux tests either through balena CI or through AppVeyor.

File: LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
+
+* Copyright 2020 Balena Ltd. under the Apache 2.0 License included in this file.
+* Copyright 2018 Zeit Inc. under the MIT License included in this file.
```
Comment (pdcastro) [inline LICENSE.md:3]: This "multiple licensing" Apache 2.0 + MIT was discussed in the architecture call of 23 Apr 2020. It satisfies the legal requirements of the original repo (https://github.com/zeit/dockerignore/blob/0.0.5/license.md) and balena's own licensing policy (Apache 2.0) and balena CI checks.

**Review by srlowe — APPROVED** [review]

**Pull request merged**